examples/SetGetBucketObjectLockConfig.java

```diff
@@ -56,7 +56,8 @@ public static void main(String[] args)
 
       System.out.println("Default retention configuration of bucket");
       System.out.println("Mode: " + bucketConfig.mode());
-      System.out.println("Duration: " + bucketConfig.duration());
+      System.out.println(
+          "Duration: " + bucketConfig.duration().duration() + " " + bucketConfig.duration().unit());
```

Comment:
Shouldn't this be `bucketConfig.duration()` instead of `bucketConfig.duration().duration()`

Reply:
`bucketConfig.duration()` returns a `RetentionDuration` object which actually contains the `duration` and `unit` i.e. days or years. 
Returning `bucketConfig.duration()`  will  just print the object which is  composed of the name of the class, an @ symbol and the hashcode of the object in hexadecimal. 
We need to fetch the exact duration and unit and thus this change is needed.

Reply:
Have `toString()` to `RetentionDurationDays` and `RetentionDurationYears` classes to have friendlier message.